### PR TITLE
Removed the Katacoda shutdown notice

### LINKED ISSUE #41836: Stale details in "Learn Kubernetes Basics" page
**This is a Bug Report**

**Problem:**
https://kubernetes.io/docs/tutorials/kubernetes-basics/ has a banner about the Katacoda shutdown. However, the tutorials that https://kubernetes.io/docs/tutorials/kubernetes-basics/ links to have been updated for local use.

**Proposed Solution:**
Remove the notice.
_Also, reword the paragraphs that refer to the Katacoda tutorial in the rest of that text._

**Page to Update:**
https://kubernetes.io/docs/tutorials/kubernetes-basics/

**Additional Information:**
/language en
/priority important-longterm

### PR DESCRIPTION
**Pull request by Raghunandan16**
fixes #41836 
image

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -9,109 +9,175 @@
...
+                version.
+              </li>
+              <li>Debug the containerized application.</li>
+            </ul>
+            <p>
+              The tutorials use Katacoda to run a virtual terminal in your web
+              browser that runs Minikube, a small-scale local deployment of
+              Kubernetes that can run anywhere. There's no need to install any
+              software or configure anything; each interactive tutorial runs
+              directly out of your web browser itself.
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/_index.html:44]: This text is wrong. Please either amend it, or change this PR to not touch it.

Comment (Raghunandan16) [reply content/en/docs/tutorials/kubernetes-basics/_index.html:44]: Do you want to have other text there or shall I simply remove any mention of katacoda on the page?

Comment (Raghunandan16) [reply content/en/docs/tutorials/kubernetes-basics/_index.html]: For now I have removed the text. Let me know if you want to add something good there

**Comment by shannonxtreme** [comment]
Im seeing a bunch of odd line breaks in the change, looks like a formatting tool broke up HTML code to meet line wrapping requirements. For example <a href=""></a> looks like

```
<a
  href=""
  ><img
     src
```

and so on. IDK how other reviewers might feel but I'd recommend reverting those types of changes here. It's okay to change the wrapping of actual text paragraphs to be <80 characters, but for stuff like links and HTML elements, leave them as-is.

**Comment by Raghunandan16** [comment]
@shannonxtreme I will fix it soon..I guess prettier did it by default..It wasn't intentional

**Comment by shannonxtreme** [comment]
@Raghunandan16 yeah I had a feeling 😅 I only found out today that I could tell VS Code to stop changing markdown numbered lists to 1. 2. 3.

**Comment by Raghunandan16** [comment]
@shannonxtreme I fixed the formatting now..Thanks for the suggestion !!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-41841--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/

File: content/en/docs/tutorials/kubernetes-basics/_index.html (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/tutorials/kubernetes-basics/_index.html]: Is it worthwhile to leave the shortcode? I don't know if this notice is still up on other pages.

Comment (Raghunandan16) [reply content/en/docs/tutorials/kubernetes-basics/_index.html]: shortcode? Sorry I didn't get you.
Are you talking about the paragraph which mentions about the "use of katacoda" in tutorials?

File: content/en/docs/tutorials/kubernetes-basics/_index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
   title: Walkthrough the basics
 ---
 
-{{% katacoda-removal %}}
```
[the remark this answers is not in the document]
Comment (Raghunandan16) [reply content/en/docs/tutorials/kubernetes-basics/_index.html:12]: Oh yeah!! Even I was thinking of the same thing while ammending it

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tutorials/kubernetes-basics/_index.html:12]: We can remove that shortcode once no English pages and no localized pages are using it.

**Review by sftim — COMMENTED** [review]
/lgtm

**Pull request merged**